samples/widgets/d3/src/runtime/widget.tsx

```diff
@@ -18,80 +18,78 @@
   A copy of the license is available in the repository's
   LICENSE file.
 */
-import { React, AllWidgetProps, jsx } from "jimu-core";
+import { jsx } from "jimu-core";
+import { useEffect, useRef } from "react";
```

Comment:
We should import react from `jimu-core`:
```
import {React} from 'jimu-core'
const {useEffect, useRef} = React
```

Reply:
Thanks - updated.